docs/book/src/clusterctl/provider-contract.md

```diff
@@ -306,15 +306,16 @@ functioning of all the `clusterctl` features both in single tenancy or multi-ten
 
 Provider authors should be aware that `clusterctl move` command implements a discovery mechanism that considers:
 
-* All the objects of Kind defined in one of the CRDs installed by clusterctl using `clusterctl init`. 
+* All the objects of Kind defined in one of the CRDs installed by clusterctl using `clusterctl init`.
 * `Secret` and `ConfigMap` objects.
-* the `OwnerReference` chain of the above objects.
+* The `OwnerReference` chain of the above objects.
+* Any object with the with the "force move" label (`clusterctl.cluster.x-k8s.io/force-move`)attached to it.
 
 `clusterctl move` does NOT consider any objects:
 
 * Not included in the set of objects defined above.
-* Included in the set of objects defined above, but not directly or indirectly to a `Cluster` object through the `OwnerReference` chain.
- 
+* Included in the set of objects defined above, but not directly or indirectly tied to a `Cluster` object through the `OwnerReference` chain (with the exception of "force move" objects mentioned above).
+
```

Comment:
TBH, IMO this section doesn't add much value as it feels like its just the opposite of what was mentioned above.

Reply:
removed this. thx.